mindsdb/integrations/handlers/mindsdb_cloud/__about__.py

```diff
@@ -0,0 +1,9 @@
+__title__ = 'MindsDB Cloud handler'
+__package_name__ = 'mindsdb_cloud_handler'
```

Comment:
Actually, the name doesn't make any change since we are not releasing them. I just checked and it is the same in all handlers, so we may decide on keeping only the name without _handler